pdfmaker/kassia.py

```diff
@@ -44,10 +45,15 @@ def setDefaults(self):
         self.pageAttrib['line_height'] = 72
         self.pageAttrib['line_width'] = self.pageAttrib['paper_size'][0] - (self.pageAttrib['left_margin'] + self.pageAttrib['right_margin'])
```

Comment:
Hmm, I'll have to think about that. It seems like page margin and space between lines of music would be the same no matter the size of the page, but maybe there are some attributes that I'm not thinking of. Also, ReportLab allows for any sized page, right? Would we just create defaults for common paper sizes?